snapcraft/cli/lifecycle.py

```diff
@@ -79,8 +94,9 @@ def _execute(  # noqa: C901
     build_environment = env.BuilderEnvironmentConfig(force_provider=provider)
     try:
         conduct_build_environment_sanity_check(build_environment.provider)
-    except MultipassMissingLinuxError as e:
-        if click.confirm(str(e)):
+    except MultipassMissingInstallableError as e:
+        click.echo("You need multipass installed to build snaps.")
```

Comment:
Googling for Multipass doesn't have it near the top. How about:

>You need Multipass to build snaps (https://github.com/CanonicalLtd/multipass).
>Would you like to install it now?

Reply:
once we have a product page

Reply:
you have it